public/app/plugins/datasource/loki/components/LokiLabelBrowser.tsx

```diff
@@ -467,7 +466,12 @@ export class UnthemedLokiLabelBrowser extends React.Component<BrowserProps, Brow
             2. Find values for the selected labels
           </Label>
           <div>
-            <Input onChange={this.onChangeSearch} aria-label="Filter expression for values" value={searchTerm} />
+            <Input
+              onChange={this.onChangeSearch}
+              aria-label="Filter expression for values"
+              value={searchTerm}
+              placeholder={'Enter a label name'}
+            />
           </div>
           <div className={styles.valueListArea}>
             {selectedLabels.map((label) => (
```

Comment:
While testing, my Loki instance went offline, so I had pre-fetched labels, but then it failed to make further requests.

With that, I saw that we display 2 messages that now are broken:

![Message 1](https://user-images.githubusercontent.com/1069378/203574920-3a8163ca-425b-4935-9c8f-802995109484.png)
![Message 2](https://user-images.githubusercontent.com/1069378/203574943-44b1299b-cdcf-42b9-ae3d-90b550b08858.png)

We need to adjust that.

To reproduce it, load the page with your Loki docker image on, and then turn it off to interact with the component.

I think moving it above the buttons would be acceptable.

![imagen](https://user-images.githubusercontent.com/1069378/203576236-a1d23b8c-4ea3-4cfe-a2e1-65f585e68416.png)

Reply:
Also I don't know what does "facetting" mean in that context at all.

Reply:
After https://raintank-corp.slack.com/archives/C03PMJLVC67/p1669214990835099 I'd suggest rename "Facetting" with "Loading" please. Thanks!